src/software/simulated_tests/simulated_play_test_fixture_test.cpp

```diff
@@ -136,7 +136,7 @@ TEST_F(SimulatedPlayTestFixtureTest,
     EXPECT_NONFATAL_FAILURE(
         runTest(terminating_validation_functions, non_terminating_validation_functions,
                 Duration::fromSeconds(0.6)),
-        "timeout duration");
+        "Waiting for timestamp of at least 0.8s");
```

Comment:
I think you might have meant this to be 0.6, not 0.8?

Reply:
no, this relates to the error message above that has 0.8s